integration-cli/docker_cli_daemon_test.go

```diff
@@ -481,6 +481,31 @@ func TestDaemonUpgradeWithVolumes(t *testing.T) {
 	logDone("daemon - volumes from old(pre 1.3) daemon work")
 }
 
+// GH#11320 - verify that the daemon exits on failure properly
+func TestDaemonExitOnFailure(t *testing.T) {
```

Comment:
@cpuguy83 because it also happens to be testing the conflict between `--bridge` and `--bip`, therefore if somebody else wanted to test that specifically, they wouldn't need to since this test does exactly that.